web/config/tailwind.config.js

```diff
@@ -29,7 +29,7 @@ module.exports = {
     },
     fontFamily: {
       sans: [
-        'Baloo 2',
+        "'Baloo 2'",
```

Comment:
```suggestion
        '"Baloo 2"',
```
I think it should be the other way round for consistency, otherwise this all looks fab!